lxdock/conf/schema.py

```diff
@@ -21,11 +21,11 @@ def get_schema():
             # The existence of the source directory will be checked!
             'source': IsDir(),
             'dest': str,
-            'set_host_acl': bool,
+            'set_host_acl': bool, # TODO: need a way to deprecate this
```

Comment:
Not really, I think we just remove it.  It was me that requested this feature, I doubt anybody else is using it.